packages/modelviewer.dev/examples/lighthouse.html

```diff
@@ -268,11 +285,13 @@ <h2>Setting up <code>&lt;model-viewer&gt;</code> for good Lighthouse scores</h2>
     body.addEventListener('keydown', onInteraction, {once:true});
 
     const onProgress = (event) => {
-      const firstBar = event.target.querySelector('.progress-bar');
-      if(event.detail.totalProgress == 1){
-        firstBar.classList.add('hide');
-      }else{
-        firstBar.style.width=`${event.detail.totalProgress*50}%`;
+      const progressBar = event.target.querySelector('.progress-bar');
+      const updatingBar = event.target.querySelector('.update-bar');
+      if (event.detail.totalProgress == 1) {
+        updatingBar.style.width=`${event.detail.totalProgress*100}%`;
+        progressBar.classList.add('hide');
+      } else {
+        updatingBar.style.width=`${event.detail.totalProgress*100}%`;
```

Comment:
Since this is happening on both sides of the conditional, let's move it out and simplify.

Reply:
done